### PR TITLE
feature: upgrade golang version to 1.12.10

### PR DESCRIPTION
**Pull request by fengzixu**
Signed-off-by: fengzixu <hnustphoenix@gmail.com>

### Ⅰ. Describe what this PR did

Due to the golang vulnerability, we need to upgrade golang version

### Ⅱ. Does this pull request fix one issue?

#989 

### Ⅳ. Describe how to verify it

I run `make build` and `make unit-test` to verify

### CODE REVIEW

**Comment by fengzixu** [comment]
@Starnop @yeya24 Request review

**Comment by tanjunchen** [comment]
@fengzixu  
image

**Comment by fengzixu** [comment]
> @fengzixu
> image

Done

**Comment by Starnop** [comment]
LGTM.

File: docs/cli_reference/dfdaemon_version.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ dfdaemon version [flags]
 dfdaemon version  0.4.1
   Git commit:     6fd5c8f
   Build date:     20190717-15:57:52
-  Go version:     go1.12.6
+  Go version:     go1.12.10
```
Comment (allencloud) [inline docs/cli_reference/dfdaemon_version.md]: Actually you do not need to update this file. But it is OK.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**